src/main/java/Bellroy/Todo.java

```diff
@@ -0,0 +1,18 @@
+package Bellroy;
+
+/**
+ * Encapsulates a Todo Task
+ */
+public class Todo extends Task {
+
+    public Todo(String description) {
```

Comment:
a very minor change, but to do should either be two words or written to-do, so a "ToDo" class is more aligned.

(Sorry, the other reviewer did an excellent job so I'm really nitpicking here)